pkg/migration/controllers/migration.go

```diff
@@ -2739,16 +2739,15 @@ func isObjectHashEqual(dynamicClient dynamic.ResourceInterface, u *unstructured.
 	return false
 }
 
-// getRelatedCRDListWRTGroupAndComponents takes input of AppRegs
-// and find out all CRDs those need to be collected.
+// getRelatedCRDListWRTGroupAndCategories takes AppRegs as input and
+// finds out all the CRDs that need to be collected.
 // The CRD list should contain
 //
```

Comment:
nit..I don't think a blank line is required here.

Reply:
`make do-fmt` is adding that line.